framework/lib/components/switch/switch-field.tsx

```diff
@@ -19,24 +32,34 @@ export const SwitchField = ({
   validate,
   onChange: onChangeCallback = identity,
   direction = 'row',
+  labelPosition = 'right',
   ...rest
 }: SwitchFieldProps) => (
   <Box w={ label ? 'full' : 'fit-content' }>
     <Field
       name={ name }
-      label={ label }
+      label=""
       isRequired={ isRequired }
       direction={ direction }
       validate={ validate }
     >
       { ({ onChange, value }) => (
-        <Switch
-          name={ name }
-          onChange={ (e) => { onChange(e); onChangeCallback(e) } }
-          value={ value }
-          data-testid="switch-field-test-id"
-          { ...rest }
-        />
+        <Flex
+          gap={ 2 }
+          direction={ labelPosition === 'left' ? 'row-reverse' : 'row' }
+        >
```

Comment:
Does this fix so we can use justify and align the label inside the switch? Right now, the label is "stuck" in a flex-start in both justify and align